src/app/views/infoblock/infoblock.js

```diff
@@ -0,0 +1,48 @@
+class InfoblockController {
+
+  /** @ngInject */
+  constructor($log) {
+    this.$log = $log;
+  }
+
+  $onChanges() {
+    if (!this.variable) {
+      return;
+    }
+
+    this.value = this.variable.value;
+    this.title = this.variable.title;
+
+    this.colorClass = '';
+    if (this.colorOverride) {
+      this.colorClass = `color-${this.colorOverride}`;
+    } else if (this.isChange && this.value !== 0) {
+      let ispositive = (this.value > 0);
+      if (this.reverseColor) {
+        ispositive = !ispositive;
+      }
+      this.colorClass = ispositive ? 'color-positive' : 'color-negative';
+    }
+
+    this.value = Math.round(this.value * 100) / 100;
```

Comment:
Trouble I had with using the number filter is I couldn't figure out how to get it to show at most 2 decimal places... by default it shows up to the number based on current locale, but if you specify the number of decimal places it always show exactly that many.

Localization for the change and percents is also a potential problem, I'm not sure if all locales always use those symbols, of the symbols always go on the right and left. Is there a better way to handle that kind of thing, or is it to just roll our own in a filter and worry about localization if it comes up?